app/src/main/java/ca/ualberta/CMPUT3012019T02/alexandria/model/BookList.java

```diff
@@ -0,0 +1,66 @@
+package ca.ualberta.CMPUT3012019T02.alexandria.model;
+
+import android.net.Uri;
+
+import ca.ualberta.CMPUT3012019T02.alexandria.R;
+
+public class BookList {
+    private int cover;
+    private String title;
+    private String author;
+    private String isbn;
+    private int status;
```

Comment:
`private int status;`
The status of the book is a String. It should be `private String status;`
Status can be "available", "requested", "accepted", or "exchanged".